src/app/public/styles/_container.scss

```diff
@@ -20,3 +20,19 @@
     width: $sky-screen-lg-min - 10;
   }
 }
+
+.sky-media-container-xs .sky-container {
+  width: auto;
+}
+
+.sky-media-container-sm .sky-container {
+  width: $sky-screen-sm-min - 10;
```

Comment:
I'm assuming you're just following a pattern here, but why are we subtracting 10 or 20 pixels for all of these `width` declarations?

Reply:
I literally just pulled these styles from above. :)